test/e2e/backup_test.go

```diff
@@ -9,75 +9,54 @@ import (
-				backupName = "backup-" + backupUUID.String()
-				restoreName = "restore-" + backupUUID.String()
-				println("backupName = " + backupName)
-				println("creating namespace " + kibishiNamespace)
-				timeoutCTX, _ := context.WithTimeout(context.Background(), time.Minute)
-				err = CreateNamespace(timeoutCTX, kibishiNamespace)
-				Expect(err).NotTo(HaveOccurred())
-
-				println("installing kibishii in namespace " + kibishiNamespace)
-				timeoutCTX, _ = context.WithTimeout(context.Background(), 30*time.Minute)
-				err = InstallKibishii(timeoutCTX, kibishiNamespace, cloudPlatform)
-				Expect(err).NotTo(HaveOccurred())
-
-				println("running kibishii generate")
-				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute*60)
-
-				err = GenerateData(timeoutCTX, kibishiNamespace, 2, 10, 10, 1024, 1024, 0, 2)
-				Expect(err).NotTo(HaveOccurred())
-
-				println("executing backup")
-				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute*30)
-
-				err = BackupNamespace(timeoutCTX, veleroCLI, backupName, kibishiNamespace)
-				Expect(err).NotTo(HaveOccurred())
-				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute)
-				err = CheckBackupPhase(timeoutCTX, veleroCLI, backupName, velerov1.BackupPhaseCompleted)
-
-				Expect(err).NotTo(HaveOccurred())
-
-				println("removing namespace " + kibishiNamespace)
-				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute)
-				err = RemoveNamespace(timeoutCTX, kibishiNamespace)
-				Expect(err).NotTo(HaveOccurred())
-
-				println("restoring namespace")
-				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute*30)
-				err = RestoreNamespace(timeoutCTX, veleroCLI, restoreName, backupName)
-				Expect(err).NotTo(HaveOccurred())
-				println("Checking that namespace is present")
-				// TODO - check that namespace exists
-				println("running kibishii verify")
-				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute*60)
-
-				err = VerifyData(timeoutCTX, kibishiNamespace, 2, 10, 10, 1024, 1024, 0, 2)
-				Expect(err).NotTo(HaveOccurred())
-
-				println("removing namespace " + kibishiNamespace)
-				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute)
-				err = RemoveNamespace(timeoutCTX, kibishiNamespace)
-				Expect(err).NotTo(HaveOccurred())
-			})
+	AfterEach(func() {
+		timeoutCTX, _ := context.WithTimeout(context.Background(), time.Minute)
+		err := client.CoreV1().Namespaces().Delete(timeoutCTX, veleroInstallOptions.Namespace, metav1.DeleteOptions{})
+		Expect(err).To(Succeed())
+	})
+	Context("When kibishii is the sample workload", func() {
+		It("should be successfully backed up and restored", func() {
+			backupName = "backup-" + uuidgen.String()
+			restoreName = "restore-" + uuidgen.String()
+			// Even though we are using Velero's CloudProvider plugin for object storage, the kubernetes cluster is running on
+			// KinD. So use the kind installation for Kibishii.
```

Comment:
This is a nit but I think using the same capitalization for names helps people understand that we are talking about the same thing. KinD, kind, Kind, we should pick one capitalization for this name and be consistent.

Reply:
The "kind" value for the parameter is used to choose a directory in https://github.com/vmware-tanzu-labs/distributed-data-generator/tree/main/kubernetes/yaml from where to deploy the test workload. Updated to use the consistent casing of `KinD` everywhere else.